.github/workflows/plugins-test.yaml

```diff
@@ -540,4 +540,28 @@ jobs:
       - name: Build the Docker image
         run: ./mvnw -f test/plugin/pom.xml clean package -DskipTests docker:build -DBUILD_NO=local >/dev/null
       - name: Run lettuce-scenario 5.x (17)
-        run: bash test/plugin/run.sh lettuce-scenario
\ No newline at end of file
+        run: bash test/plugin/run.sh lettuce-scenario
+  PlayFramework:
+    runs-on: ubuntu-18.04
+    timeout-minutes: 90
+    strategy:
+      fail-fast: true
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: true
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Build SkyWalking Agent
+        run: ./mvnw clean package -DskipTests -Pagent >/dev/null
+      - name: Build the Docker image
+        run: ./mvnw -f test/plugin/pom.xml clean package -DskipTests docker:build -DBUILD_NO=local >/dev/null
+      - name: Run Play! Framework
+        run: bash test/plugin/run.sh play-scenario
```

Comment:
As it runs fast enough, may be it should combined with lettuce.  what about you ? @wu-sheng

Reply:
Yes, PlayFramework test cases should be merged into Lettuce group.

Reply:
@wayilau @wu-sheng Your mean is add PlayFramework test to LETTUCE job as a step ?

Reply:
@wu-sheng @wayilau Can i merge LETTUCE and PlayFramework into Ehcache_Undertow_Jedis job ?

Reply:
@aiyanbo execution time should less than 35m, i think put LETTUCE and Play  together is more better.

Reply:
After you add play to lettuce, please rename LETTUCE to LETTUCE_PlayFramework. @aiyanbo

Reply:
> After you add play to lettuce, please rename LETTUCE to LETTUCE_PlayFramework. @aiyanbo

👌

Reply:
@wu-sheng @wayilau I was merged Play into LETTUCE, but can i merged Play and LETTUCE into Ehcache_Undertow_Jedis?